code/game/machinery/machinery.dm

```diff
@@ -246,6 +246,12 @@ Class Procs:
 		return
 	if(!isturf(M.loc) && M.loc != src)
 		return
+	if(getBrainLoss() >= 60)

```

Comment:
Moving this to human/canUseTopic() might be bad, because people have used canUseTopic outside of just checking for Topic because it was such a nice general-use "Can this person use this thing?" check.